baseapp-payments/testproject/settings.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+
+from baseapp_core.settings.env import env
+from baseapp_core.tests.settings import *  # noqa
+
+# import from source code dir
+here = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(0, here)
+sys.path.insert(0, os.path.join(here, os.pardir))
+
+SITE_ID = 300
+
+DEBUG = True
```

Comment:
```suggestion
```
We can remove DEBUG variable from here since we are importing from baseapp_core/tests/settings.py